kythe/cxx/indexer/cxx/testdata/basic/scopes.cc

```diff
@@ -0,0 +1,4 @@
+// TODO(justinbuchanan): fix indexer to include leading '(' in lambda anchor
+//- @") {}" defines Lambda
```

Comment:
Since  lambda is syntactically introduced by the capture list (`[]`), the definition should probably span the entire thing.  The parens are entirely optional here; `[]{}` is a valid lambda definition.

Reply:
Done.